docs/run-platform-server/configuring.mdx

```diff
@@ -0,0 +1,145 @@
+---
+title: Configuring
+sidebar_position: 30
+---
+
+import { Alert } from "@site/src/components/Alert";
+import { CodeExample } from "@site/src/components/CodeExample";
+
+Pre-requisites:
+
+- [Installation](./installing.mdx) is completed on host machine, it should have two executables, `stellar-horizon` and `stellar-core`. Per the installation instructions, `stellar-core` should be included in your host o/s PATH as these configurations will result in Horizon searching for `stellar-core` to execute.
+- [Prepare the Database if first time](#prepare-the-database-first-time)
+
+You are now ready to setup configuration parameters to perform three important, distinct roles:
+
+- **serving requests** a regular web-based HTTP API,
+- **ingesting ledgers** from core nodes of a Stellar network to keep its world-view up to date, and
+- **transaction submission** for sending transactions to the Stellar network.
+
+You choose from one of two following deployment modes to perform the roles and proceed with configuration parameters to that means. A note on configuration parameters, the same parameter can be provided either as command line argument or an environment variable.
```

Comment:
I find the "configuration parameters to that means" wording a bit confusing. Here's a suggested reword, plus moving the note to its own "Info" block:

```suggestion
To perform these roles, you can choose from one of two of the deployment modes that follow. Each has its own configuration parameters.

<Alert>

In all configuration examples, the same parameter can be provided either as command line argument or an environment variable. If both are provided, the command line takes precedence.

</Alert>

```